contracts/implementation/PoolFactory.sol

```diff
@@ -102,9 +102,7 @@ contract PoolFactory is IPoolFactory, Ownable {
             abi.encode(
                 deploymentParameters.leverageAmount,
                 deploymentParameters.quoteToken,
-                deploymentParameters.oracleWrapper,
-                deploymentParameters.updateInterval,
-                deploymentParameters.frontRunningInterval
+                deploymentParameters.oracleWrapper
             )
```

Comment:
I don't understand why we have to or benefit from removing this.

Reply:
removing what?

Reply:
I think this is referencing @kumar-ish comments on `updateInterval` and `frontRunningInterval`? Not too sure though.